src/platform/plugins/shared/data/common/search/expressions/esql.ts

```diff
@@ -392,7 +398,15 @@ export const getEsqlFn = ({ getStartDependencies }: EsqlFnArguments) => {
           }
           const columnNames = updatedWithVariablesColumns?.map(({ name }) => name);
 
-          const rows = body.values.map((row) => zipObject(columnNames, row));
+          const rows = body.values.map((row) =>
```

Comment:
I understand this change but was the previous format causing any problems? Because it had the advantage of returning a smaller response

Reply:
I'm close, sorry for the late work here, but is not high priority. I'm pushed an update to the parallel PR to test one last thing and we should be good to go

Reply:
Ok I've removed that logic and removed also the validation on empty charts done in some chart type (that needs to verify that all the columns are empty to consider a chart empty) done here https://github.com/elastic/kibana/pull/233369/commits/3ace7c0c0f61e0de415b2d814ca3e5b0f8cc6cf8